bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -302,6 +311,23 @@ private void scheduleInitialAccess(BoschHttpClient httpClient) {
         }
     }
 
+    private void updateThingProperties() {
+        try {
+            PublicInformation publicInformation = getPublicInformation();
+            @Nullable
+            Map<String, String> properties = new HashMap<>();
+            properties.put(THING_PROPERTY_MAC_ADDRESS, publicInformation.macAddress);
+            properties.put(THING_PROPERTY_IP_ADDRESS, publicInformation.shcIpAddress);
```

Comment:
This is already a configuration parameter, so it's redundant to also provide this as a property.

Reply:
Good point, I will remove it 👍

Reply:
Since this was added in previous versions (I just replaced the string literal with a constant), I decided to add some code that removes the  redundant `ipAddress` thing property for existing things. Let me know if you agree whether this is the correct approach.